<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resources.qrc">
    <normaloff>:/main_icon</normaloff>:/main_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="circularProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border-radius: 150px;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.749 rgba(199, 200, 188, 0), stop:0.75 	rgba(106, 200, 104, 255));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularBg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius:150px;
	background-color: rgba(199, 200, 188, 120);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="container">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>270</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border-radius:135px;
	background-color: rgb(85, 159, 83);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>195</width>
        <height>185</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTitle">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#79e276;&quot;&gt;DNS SHOP&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#6ac868;&quot;/&gt; PARSER&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPercentage">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: none;
	margin-left:30px;
	margin-right:30px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-size:48pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCredits_2">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-radius:10px;
	background-color: rgb(109, 202, 105);
	margin-left: 30px;
	margin-right:30px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>loading...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelCredits">
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(120, 223, 116);
	background-color: none;
	margin-left:30px;
	margin-right:30px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>by skhe55</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>circularBg</zorder>
    <zorder>circularProgress</zorder>
    <zorder>container</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
